modules/persistent-storage-local-tolerations.adoc

```diff
@@ -40,9 +40,17 @@ To configure local volumes for scheduling on tainted nodes:
       - key: localstorage <1>
         operator: Equal <2>
         value: "localstorage" <3>
+    storageClassDevices:
+        - storageClassName: "localblock-sc"
+          volumeMode: Block <4>
+          devicePaths: <5>
+            - /dev/xvdg
+
 ----
 <1> Specify the key that you added to the node.
 <2> Specify the `Equal` operator to require the `key`/`value` parameters to match. If operator is 'Exists`, the system checks that the key exists and ignores the value. If operator is `Equal`, then the key and value must match.
 <3> Specify the value `local` of the tainted node.
+<4> The volume mode, either Filesystem or Block, defining the type of the local volumes.
+<5> The path to where the local volumes have been attached to the node.
```

Comment:
For the <5>, `A list of devices which would be chosen for local storage.` looks better.

Reply:
> For the <5>, `A list of devices which would be chosen for local storage.` looks better.

@qinpingli Does this look okay?
`The path containing a list of local storage devices to choose from.`

If so, could I please get a LGTM? Thanks!